### DISCUSSION TITLE
Node.js 22 container image

### DISCUSSION THREAD

**Discussion by prabhu** [SUSE/bci#19] [General]
Hello,

Any plans to offer Node.js 22 container images? We are currently using nvm to install version 22, but prefer a bci image over this.

**Comment by dcermak** [comment]
We have a nodejs 22 container based on openSUSE Tumbleweed at the moment. BCI Images based on SLES with nodejs 22 will be available once node 22 hits SLES, which will take a bit (unfortunately, I cannot give you a concrete ETA)

**Reply by prabhu** [reply]
Happy to try the tumbleweed version. Can you share the reference?

**Reply by dcermak** [reply]
Apologies, I mixed it up with the openJDK 22 image. We don't have nodejs 22 in any codestream yet and it will probably take a few weeks for it to become available.

**Reply by dirkmueller** [reply]
work is in progress of adding it to tumbleweed. we'll add it when available there. it is going to be available for pull from `registry.opensuse.org/opensuse/bci/nodejs:22` when published and listed on https://artifacthub.io/packages/container/opensuse-bci-node/node as usual.
